lib/matplotlib/axes/_base.py

```diff
@@ -3326,7 +3326,8 @@ def tick_params(self, axis='both', **kwargs):
         Change the appearance of ticks, tick labels, and gridlines.
 
         Tick properties that are not explicitly set using the keyword
-        arguments remain unchanged unless *reset* is True.
+        arguments remain unchanged unless *reset* is True. For the current
+        style defaults, see :meth:`matplotlib.axis.Axis.get_tick_params`.
```

Comment:
```suggestion
        style defaults, see `.Axis.get_tick_params`.
```
we can shorten this, which makes the plain docstring more readable. See also https://matplotlib.org/devdocs/devel/documenting_mpl.html#referring-to-other-code.

Reply:
This doesn't seem to work in the [generated docs](https://output.circle-artifacts.com/output/job/58a01a40-e3df-4d10-a3a1-57366c92e770/artifacts/0/doc/build/html/api/_as_gen/matplotlib.axes.Axes.tick_params.html#matplotlib.axes.Axes.tick_params).